hackclub.com.yaml

```diff
@@ -1656,9 +1656,30 @@ nust:
     type: CNAME
     value: nusthackclub.netlify.app.
 ocsa:
-  - ttl: 1
+  - ttl: 3600
     type: TXT
-    value: google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+    values: 
+    -google-site-verification=tk_snQuv19iUwoOlsbKZv2Wwg5stfcusEm1Q8NxDiVA
+    -v=spf1 include:_spf.google.com ~all
```

Comment:
should it be `- v=spf1`? and also `- google-site-...`?

Reply:
i think there should be space between the `-` and the value

Reply:
done!